tests/test_filter_distribcell/case-1/results_true.dat

```diff
@@ -0,0 +1,14 @@
+k-combined:
```

Comment:
It looks like all the same values just placed in a different order.  If you want, I'll try to work out what the exact difference is.